docs/sphinx/snippets/cpp/using/backends/dynamics.cpp

```diff
@@ -0,0 +1,141 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2025 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+#include "cudaq/algorithms/evolve.h"
+#include "cudaq/dynamics_integrators.h"
+#include "cudaq/operators.h"
+#include <cmath>
+#include <complex>
+#include <cudaq.h>
+#include <iostream>
+#include <vector>
+
+using namespace cudaq;
+
+int main() {
+  // [Begin Transmon]
+  // Parameters
+  double omega_z = 6.5;
```

Comment:
should we add these values to the Python tab as well? 
Currently it doesn't have the specific values.

Reply:
Yea, I think it's a good idea to keep the Python tab and C++ tab in sync.